app/assets/stylesheets/components/inline-form.scss

```diff
@@ -0,0 +1,3 @@
+.inline-form {
```

Comment:
I reckon we can do this without the need for an additional SCSS file and class. Worth speaking to @titlescreen about how to do that?